### DISCUSSION TITLE
No RSS websites support

### DISCUSSION THREAD

**Discussion by mkagit** [Athou/commafeed#1455] [Ideas]
Nowadays a lot of blog and changelog websites doesn't support RSS. There is apps like Feedly and Inoreader that support to create a RSS feed from a **no** RSS website , but it is a premium paid feature. Would be great if commafeed could support a feature it.

**Comment by Athou** [comment]
I'm sorry but this is not planned for CommaFeed. There are apps dedicated to this like [RSSHub](https://github.com/DIYgod/RSSHub) or [rss-bridge](https://github.com/RSS-Bridge/rss-bridge) that are doing a better job than I ever could :)

**Comment by mkagit** [comment]
Fine! I'll look for these projects

Em qui., 20 de jun. de 2024 às 08:35, Jérémie Panzer <
***@***.***> escreveu:
